static/js/lib/api/api.js

```diff
@@ -7,52 +7,26 @@ import qs from "query-string"
 import { PATCH, POST, DELETE } from "redux-hammock/constants"
 import { fetchJSONWithCSRF } from "redux-hammock/django_csrf_fetch"
 
-import { buildSearchQuery } from "./search"
+import { buildSearchQuery } from "../search"
 import {
   fetchJSONWithAuthFailure,
   fetchWithAuthFailure,
   fetchJSONWithToken
 } from "./fetch_auth"
-import { toQueryString } from "../lib/url"
-import { getPaginationSortParams } from "../lib/posts"
-import { objectToFormData } from "../lib/forms"
+import { getCommentSortQS } from "./util"
 
-import type { AuthResponse, AuthFlow } from "../flow/authTypes"
+import type { AuthResponse, AuthFlow } from "../../flow/authTypes"
 import type {
-  Channel,
-  ChannelContributors,
-  ChannelModerators,
-  Contributor,
-  Moderator,
-  Subscriber,
   GenericComment,
-  CreatePostPayload,
-  PostListPaginationParams,
-  Post,
   CommentFromAPI,
   MoreCommentsFromAPI,
-  GenericReport,
-  ReportRecord,
   Profile,
   ProfilePayload,
-  SocialAuth,
-  PostListResponse
-} from "../flow/discussionTypes"
-import type { NotificationSetting } from "../flow/settingsTypes"
-import type { WidgetListResponse } from "../flow/widgetTypes"
-import type { EmbedlyResponse } from "../reducers/embedly"
-import type { SearchParams } from "../flow/searchTypes"
-
-const paramsToQueryString = paramSelector =>
-  R.compose(
-    toQueryString,
-    R.reject(R.isNil),
-    paramSelector
-  )
-
-const getPaginationSortQS = paramsToQueryString(getPaginationSortParams)
-
-const getCommentSortQS = paramsToQueryString(R.pickAll(["sort"]))
+  SocialAuth
+} from "../../flow/discussionTypes"
+import type { NotificationSetting } from "../../flow/settingsTypes"
+import type { WidgetListResponse } from "../../flow/widgetTypes"
+import type { SearchParams } from "../../flow/searchTypes"
 
 export function search(params: SearchParams): Promise<*> {
```

Comment:
Was this function, `getComments` and other functions left in this file because they didn't seem to fit elsewhere?

Reply:
Sounds good